src/GlobalPayments.Api/Terminals/INGENICO/Interfaces/TcpListenerEx.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net;
+using System.Net.Sockets;
+
+namespace GlobalPayments.Api.Terminals.Ingenico {
+    public class TcpListenerEx : TcpListener {
+        public TcpListenerEx(IPAddress localaddr, int port) : base(localaddr, port) {
+
+        }
+
+        public new bool Active { get { return base.Active; } }
```

Comment:
I assume the purpose of this new class is that Active is not publically available on the existing TcpListener?

Reply:
Yes, correct Active property is not publically available in TcpListener.